Bottom: 75c2527b2977c296cdc144568b5058383a087927
Top:    04158136d70df125fdd0ef9a3063ffb0b737a115
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 01:47:13 +0100

Cure crosshair lagging on screen

FIXME: SHOULD NOT RE-WRITE THE GTK HID


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index d3d5c16..37dd2af 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -674,9 +674,7 @@ ghid_port_window_motion_cb (GtkWidget * widget,
   queue_tooltip_update (out);
 #endif
 
-  ghid_show_crosshair (FALSE);
-  ghid_show_crosshair (TRUE);
-  if (moved && have_crosshair_attachments ())
+  if (moved)
     ghid_draw_area_update (gport, NULL);
   return FALSE;
 }
